### ISSUE TITLE
Is it possible to programmatically clear the divi css cache?

### ISSUE THREAD

**Issue by wpt00ls** [elegantthemes/create-divi-extension#387]
### Problem Description

Is it possible to clear the divi css cache programmatically? 

I have a free & pro plugin, **The pro plugin adds some custom styles via divi module to contact form 7**. 

When the pro plugin is disabled or when it's trial expires (and the user has the free version only), **the styles added via the pro plugin still presists as they are stored in the divi css cache.** 

**I'd like to invalid the divi css cache programmatically** when my pro version trial expires. 

Is this possible? Can some please point me in the right direction?

Much appreciated!

Regards

**Comment by lanresmith** [comment]
Yes, you can use `ET_Core_PageResource::remove_static_resources` from the file `core/components/PageResource.php`.

**Comment by Sumit89612** [comment]
@lanresmith but this function remove all the css added by Visual Builder for first time loading and for second time loading it is working perfectly.
Any suggestion for this ?
